Geometry/HGCalCommonData/interface/HGCalGeomParameters.h

```diff
@@ -118,9 +118,17 @@ class HGCalGeomParameters {
   void resetZero(std::vector<double>&);
 
   constexpr static double tan30deg_ = 0.5773502693;
+  constexpr static int philippeEE = 2;
+  constexpr static int philippeHE = 3;
+  constexpr static int katjaFile = 4;
```

Comment:
It's a bit odd to embed people's names into the source code. It means if you don't know Katja, then you won't understand this variable. I presume these names are temporary during HGCal design and will be replaced in the near future.

Reply:
Changed